apps/hash-frontend/src/blocks/use-fetch-block-subgraph.ts

```diff
@@ -106,7 +106,7 @@ export const useFetchBlockSubgraph = () => {
           ],
           vertices: {
             [placeholderEntity.metadata.recordId.entityId]: {
-              [now as EntityRevisionId]: {
+              [now]: {
```

Comment:
Was this ESLint that removed this or did you? I just tried it locally and I'm surprised to see that it's not required, I thought it would be (maybe because it's cast-able so it's happening implicitly..).

If it was manually removed, I think it's nicer as it was though, it's more explicit/correct.